### ISSUE TITLE
can "Person" be an IETF WG / mailing list?

### ISSUE THREAD

**Issue by rohanmahy** [ietf-wg-mediaman/6838bis#9]
Some IETF specs (ex: [RC9594](https://www.rfc-editor.org/rfc/rfc9594.html) use an IETF WG name and mailing list for the Person and Email, but most do not (I saw one that referred IANA to the Editor's Addresses). I would like this document to either tell WGs explicitly "DO THAT" or "DON'T DO THAT" please.

WGs come and go and hopefully live shorter lives than people, but IETF mailing lists can last a pretty long time.

**Comment by alvestrand** [comment]
Personally, I'm skeptical of MLs as change controllers.
I believe the IESG (or successor bodies) will live longer than most WGs.
Mailing lists are frequently closed when the WG is gone and the maintainer gets tired of dealing with the spam.
Also, mailing lists have no built-in decision procedure - how can you tell if a mailing list with 100+ mostly silent members agree on something?

**Comment by martinthomson** [comment]
This isn't about change controllers.  There are three relevant fields in the template:

*   Person and email address to contact for further information:
*   Author:  
*   Change controller: 

Recent registrations that I've created use "IETF" for the last and the document authors for the first two.  Rohan is interested in whether the WG mailing list is a better place to point people for the first (I think).  I'd also ask about the second for WG products.

**Comment by rohanmahy** [comment]
Yes, as Martin said I am referring to the first "Person & email address to
contact for further information:"
thanks,
-rohan

**Comment by alvestrand** [comment]
OK, changed the title. In that case, I think an IETF (or other org's) mailing list makes sense; it would be the change controller's responsibility to update the field when a mailing list shuts down.

**Comment by rohanmahy** [comment]
@mnot as the author do you want to weigh in on this? If you are amenable, I'll send you a short PR. Maybe instead of "Person and email address to contact for further information", it would be clearer: "Contact name and email address for further information". Then the text could explain that Contact name is an IETF/IRTF mailing list if the work is a product of the IETF, or the name/email of a human person otherwise?

And you might as well give your views about the "Author" field as well.

**Comment by mnot** [comment]
I think it's reasonable, but keeping it up to date requires action, and it's often not clear who the action holder is. Somewhat related, I'd like to do some work on clarifying what the responsibilities and powers of a designated expert are.

**Comment by alvestrand** [comment]
Formally, I think the change controller needs to be responsible for updating the "Person to contact for further info".
I don't like encumbering the DE with too much proactive responsibility - we might ask IANABIS whether they should maintain a database of contact addresses and take the responsibility of flushing out contacts that don't work any more.

**Comment by polx** [comment]
Assuming the media-types mailing-list is still alive, isn't the procedure used in the media-type declaration (but with a different template) to post an actualization of the contact-addresses at the media-types mailing-list then handling by directors the right method?

The same "authority inspection" worries are concerned here, or? And making it transparent is publicly important for sure.

**Comment by alvestrand** [comment]
This (update procedure) seems to be a more generic question, pertaining to section 6.1 "change procedures". I filed #21 to track that question.

**Comment by mnot** [comment]
I'm not convinced that a fallback pointer will be any more reliable, and it adds complexity to the procedure and the registry. Also, all existing entries won't have a fallback, limiting the utility of this approach for some time and necessitating a fallback procedure for the absence of a fallback. 

I'd suggest that either: 

1. The expert be given explicit power to chase up unresponsive contacts and update using reasonable measures, with a fallback to the IESG if there's a dispute of their decision.
2. We close this issue without action; it's working well enough.

**Comment by mnot** [comment]
Note also that the change section already has:

> The IESG may reassign responsibility for a media type. The most common case of this will be to enable changes to be made to types where the author of the registration has died, fallen out of contact, or is otherwise unable to make changes that are important to the community.

**Comment by mnot** [comment]
Regarding the _original_ issue here, the standards tree section already says:

> The change controller of a media type registered in the standards tree is assumed to be the standards-related organization itself.

We could refine that with something like:

> In the case of IETF standards, the change controller is normally the IESG.

**Comment by polx** [comment]
I am missing the transparency of the operation when the IESG or the experts change reassign. It is probably in some by-laws of the IESG but including that at least the conclusion of a reassignment is visible on the ietf-types mailing-list would help trust, I think. One could adapt the current sentence for this, I think.

**Comment by mnot** [comment]
Experts are listed in the registry, e.g., https://www.iana.org/assignments/media-types/media-types.xhtml

If we want to have norms/rules for when/how they're announced, I think that should be done in IANABIS, not as a one-off here.

**Comment by polx** [comment]
I am not making in question the list of experts. I proposing adjust the sentence so that a reassignment of responsibility for a media-type is an action that is transparently relayed to some public place such as the mailing-list.

**Comment by mnot** [comment]
I understand. I stand by my comment.
